### DISCUSSION TITLE
Adding Custom fields

### DISCUSSION THREAD

**Discussion by emidiao** [walinejs/waline#1492] [Q&A] [answered]
is there a way to add custom fields  (e.g. a checkbox) to the comment form or to the user registration form?
Sorry if already asked

**Comment by lizheming** [comment]
What are your original needs? We doesn't support to add custom field into registration form right now. If you want to do it, you need custom registration ui with Waline API.

**Comment by emidiao** [comment, marked as answer]
I'd Need to add a couple of boolean fields to the user profile, basically acceptance of Tos and privacy policy

**Reply by lizheming** [reply]
I think it's better add the policy tips into comment box of waline. Because user's privacy data will be inserted when comment posted, not user registration. so I think it's better to add the policy tips into placeholder? Or you can ref https://github.com/orgs/walinejs/discussions/1485 to stop store privacy data to avoid privacy problem.
